tests/console/copy_iso_to_external_drive.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Copy the installation ISO to an external drive
+# Maintainer: Joachim Rauch <jrauch@suse.com>
+# Tags: boo#1040749
+
+use base 'btrfs_test';
+use strict;
+use testapi;
+
+sub run() {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # choose unpartioned disk and set <$disk> shell variable
+    $self->set_unpartitioned_disk_in_bash;
+
+    #partition HDD2
+    script_run 'echo "Disk: $disk"';
```

Comment:
In my opinion this would be overkill for a debug output, since it has nothing to do with the testing itself and increases the risk of failing when qemu loses chars